Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/profile_page.tsx

```diff
@@ -38,20 +42,756 @@ interface Properties {
   onSubmitPassword?: (password: string) => void;
 }
 
+interface State {
+  countryDatabase: Nexus.CountryDatabase;
```

Comment:
Can't use this as a state. If you need this then pass it in as a prop.

Reply:
What about the group name? Because having it be a prop on it's own seems weird but it's not included in identity.